tests/ldevnet.go

```diff
@@ -27,14 +27,15 @@ func LaunchDevnetDocker(t *testing.T, numMiners int, ipfsMaddr string, mountVolu
 		devnetEnv("NUMMINERS", strconv.Itoa(numMiners)),
 		devnetEnv("SPEED", "500"),
 		devnetEnv("IPFSADDR", ipfsMaddr),
+		devnetEnv("BIGSECTORS", false),
```

Comment:
This change is because now default this was switched to true in `lotus-devnet`, so not specifying a value would be the opposite as before.